backend/src/router/block/get-hash-by-id.ts

```diff
@@ -13,10 +13,12 @@ export const router = trpc.router<Context>().query("getHashById", {
     let selection = indexerDatabase
       .selectFrom("blocks")
       .select("block_hash as hash");
-    if (isNaN(parseInt(blockId))) {
+    if ([43, 44].indexOf(blockId.length) >= 0) {
       selection = selection.where("block_hash", "=", blockId);
-    } else {
+    } else if (!isNaN(parseInt(blockId))) {
```

Comment:
This will search for block `1account.near`, and those accounts are valid.

Reply:
What if we use `Number`? So the condition will be `if (!isNaN(Number(blockId)))`

Reply:
LGTM